src/main/java/com/jalasoft/jfc/model/video/VideoConverter.java

```diff
@@ -139,4 +141,24 @@ public String getThumbnail(Param param) throws CommandValueException {
             throw new CommandValueException(cve.getMessage(), this.getClass().getName());
         }
     }
+
+    /**
+     * Zips a list of files.
+     * @param videoParam receives videoParam.
+     * @throws IOException when is a invalid file path.
+     */
+    private void zipFile(VideoParam videoParam) throws IOException {
+        ZipFolder zip = new ZipFolder();
+
+        final String BACKSLASH = "/";
+        final String ZIP_TAG = ".zip";
```

Comment:
if you are using this variable only in this class, you can declare as private variables